Rlog/View/WorkSpace/WorkSpaceCreateScheduleListView.swift

```diff
@@ -35,24 +36,28 @@ struct WorkSpaceCreateScheduleListView: View {
         .padding(.horizontal)
         .toolbar {
             ToolbarItem(placement: .navigationBarTrailing) {
-                    toolbarNextButton
+                toolbarNextButton
             }
         }
         .sheet(isPresented: $viewModel.isShowingModal) {
-            WorkSpaceCreateCreatingScheduleView()
+            WorkSpaceCreateCreatingScheduleView(isShowingModal: $viewModel.isShowingModal, scheduleList: $viewModel.scheduleList)
```

Comment:
```suggestion
            WorkSpaceCreateCreatingScheduleView(
            isShowingModal: $viewModel.isShowingModal, 
            scheduleList: $viewModel.scheduleList
            )
```

긴 코드는 파라미터 정리해주시면 읽기 좋더라고요

Reply:
넵! 적용했습니다!